certification/internal/shell/has_required_labels.go

```diff
@@ -59,7 +59,7 @@ func (p *HasRequiredLabelsCheck) Metadata() certification.Metadata {
 
 func (p *HasRequiredLabelsCheck) Help() certification.HelpText {
 	return certification.HelpText{
-		Message:    "Container images must include the following metadata: name, vendor, version, release, summary, description",
+		Message:    "Check Check HasRequiredLabel encountered an error. Please review the preflight.log file for more information. encountered an error. Please review the preflight.log file for more information.",
```

Comment:
This looks like a duplicate paste

Reply:
oops looks like I did double paste good catch